pkgs/top-level/python-packages.nix

```diff
@@ -6883,7 +6883,7 @@ self: super: with self; {
 
   libtmux = callPackage ../development/python-modules/libtmux { };
 
-  libtorrent-rasterbar = (toPythonModule (pkgs.libtorrent-rasterbar.override { inherit python; })).python;
```

Comment:
is the python3 = python; override still necesarry?

Reply:
Probably not, I can remove it.

Reply:
On that note, should I also explicitly mark ` libtorrent-rasterbar-1_2_x` as disabled for 3.12 and up? (Couldn't figure out how to fix it, and the 2_0_x patch didn't do it.)